doo-swagger.json

```diff
@@ -5925,6 +6206,259 @@
         "data",
         "_links"
       ]
+    },
+    "WebhookCollectionPostRequest": {
+      "type": "object",
+      "properties": {
+        "user": {
+          "type": "string",
+          "required": true
```

Comment:
required fields should be specified on each object on `required` string array with specified all the required fields on this object (please check existing endpoints)

Each object should have `required` property! Except the case when all the fields in the object can be without value